prow/plugins/assign/assign.go

```diff
@@ -140,6 +146,13 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	if len(assign) > 0 {
 		log.Printf("Adding assignees to %s/%s#%d: %v", ae.org, ae.repo, ae.number, assign)
 		if err := gc.AssignIssue(ae.org, ae.repo, ae.number, assign); err != nil {
+			if _, ok := err.(*github.MissingUsers); ok {
```

Comment:
You can use the first return value of the type assertion.
```go
if mu, ok := err.(*github.MissingUsers); ok {
    // mu is now a string slice, []string.
    msg := fmt.Sprintf("GitHub didn't allow me to assign the following users: %s.\n\nNote that only %s members can be assigned.", strings.join(mu, ", "), ae.org)
    ...
}
```
This allows us to avoid stringly logic involving error strings, like `err.Error()[len("cannot assign "):]`. That is something you should never ever do, and it causes all sorts of problems for kubernetes.